vms/platformvm/state/state.go

```diff
@@ -61,35 +61,35 @@ var (
-	pendingPrefix                       = []byte("pending")
-	validatorPrefix                     = []byte("validator")
-	delegatorPrefix                     = []byte("delegator")
-	subnetValidatorPrefix               = []byte("subnetValidator")
-	subnetDelegatorPrefix               = []byte("subnetDelegator")
-	nestedValidatorWeightDiffsPrefix    = []byte("validatorDiffs")
-	nestedValidatorPublicKeyDiffsPrefix = []byte("publicKeyDiffs")
-	flatValidatorWeightDiffsPrefix      = []byte("flatValidatorDiffs")
-	flatValidatorPublicKeyDiffsPrefix   = []byte("flatPublicKeyDiffs")
-	txPrefix                            = []byte("tx")
-	rewardUTXOsPrefix                   = []byte("rewardUTXOs")
-	utxoPrefix                          = []byte("utxo")
-	subnetPrefix                        = []byte("subnet")
-	subnetOwnerPrefix                   = []byte("subnetOwner")
-	transformedSubnetPrefix             = []byte("transformedSubnet")
-	supplyPrefix                        = []byte("supply")
-	chainPrefix                         = []byte("chain")
-	singletonPrefix                     = []byte("singleton")
-
-	timestampKey      = []byte("timestamp")
-	currentSupplyKey  = []byte("current supply")
-	lastAcceptedKey   = []byte("last accepted")
-	heightsIndexedKey = []byte("heights indexed")
-	initializedKey    = []byte("initialized")
-	prunedKey         = []byte("pruned")
+	BlockIDPrefix                       = []byte("blockID")
+	BlockPrefix                         = []byte("block")
+	ValidatorsPrefix                    = []byte("validators")
+	CurrentPrefix                       = []byte("current")
+	PendingPrefix                       = []byte("pending")
+	ValidatorPrefix                     = []byte("validator")
+	DelegatorPrefix                     = []byte("delegator")
+	SubnetValidatorPrefix               = []byte("subnetValidator")
+	SubnetDelegatorPrefix               = []byte("subnetDelegator")
+	NestedValidatorWeightDiffsPrefix    = []byte("validatorDiffs")
+	NestedValidatorPublicKeyDiffsPrefix = []byte("publicKeyDiffs")
+	FlatValidatorWeightDiffsPrefix      = []byte("flatValidatorDiffs")
+	FlatValidatorPublicKeyDiffsPrefix   = []byte("flatPublicKeyDiffs")
+	TxPrefix                            = []byte("tx")
+	RewardUTXOsPrefix                   = []byte("rewardUTXOs")
+	UTXOPrefix                          = []byte("utxo")
+	SubnetPrefix                        = []byte("subnet")
+	SubnetOwnerPrefix                   = []byte("subnetOwner")
+	TransformedSubnetPrefix             = []byte("transformedSubnet")
+	SupplyPrefix                        = []byte("supply")
+	ChainPrefix                         = []byte("chain")
+	SingletonPrefix                     = []byte("singleton")
+
+	TimestampKey      = []byte("timestamp")
+	CurrentSupplyKey  = []byte("current supply")
+	LastAcceptedKey   = []byte("last accepted")
+	HeightsIndexedKey = []byte("heights indexed")
+	InitializedKey    = []byte("initialized")
+	PrunedKey         = []byte("pruned")
```

Comment:
just checking if these were also exported intentionally.

Reply:
yeah - I intentionally exported all of the key constants.